app/pdf.js

```diff
@@ -4,83 +4,50 @@
 
 'use strict'
 
-const electron = require('electron')
-const config = require('../js/constants/config')
-const BrowserWindow = electron.BrowserWindow
-
-const renderUrlToPdf = (appState, action) => {
-  let url = action.url
-  let savePath = action.savePath
-  let openAfterwards = action.openAfterwards
+const tabs = require('./browser/tabs')
+const {fileUrl} = require('../../js/lib/appUrlUtil')
```

Comment:
path should be `../js/lib/appUrlUtil`

Reply:
pushed out a fix for that to this branch